saleor/graphql/product/schema.py

```diff
@@ -182,6 +182,10 @@ class ProductQueries(graphene.ObjectType):
         Category,
         id=graphene.Argument(graphene.ID, description="ID of the category."),
         slug=graphene.Argument(graphene.String, description="Slug of the category"),
+        updatedAt=graphene.Argument(
```

Comment:
```suggestion
        updated_at=graphene.Argument(
```

Reply:
The field has been removed after the review from @IKarbowiak